docs/Lua API.rst

```diff
@@ -2984,6 +2984,63 @@ Predefined instance methods:
 
 To avoid confusion, these methods cannot be redefined.
 
+.. _custom-raw-tags:
+
+custom-raw-tags
+===============
+
+A module for reading custom tags added to the raws by mods.
```

Comment:
Sorry for my ignorance on this, but can this module be used to read vanilla (non-mod-added) tags in the raws? Is it useful to be able to read vanilla tags? Do vanilla tags even exist?

Reply:
It can actually read vanilla tags, I've been using CASTE_NAME for a lot of my testing. Is it useful (otherwise) to read vanilla tags? No, I highly doubt it. The "raw struct" should have all the information already outside of its raw strings. And I don't really understand the question "do vanilla tags even exist?" lol, vanilla tags are ones that are used by the game and don't throw errors.

Reply:
I'm not really sure if there was a change you wanted me to make, but, `A module for reading custom tags added to the raws by mods. Can also read vanilla tags if requested.` is this more on-point?

Reply:
I meant, are tags just a mod thing or are they used by vanilla as well, but I understand from your response that tags are common in the raws, it's just that the information from those tags are already accessible in c++ and lua via the generated struct definitions, so it's not likely that anyone would need to use this module to read vanilla tags.  No need to add anything about vanilla tags to the docs, then.